README.md

```diff
@@ -9,12 +9,16 @@ Updating...
 
 Currently, there are two types of simulation adapters we support: DiRA and UIT. To choose different adapters, you should add `adapter` argument (please see [bin/node.py](bin/node.py)).
 
-**Select Adapters**:
-- `adapter`: the value of `adapter` argument must be in `[dira, uit]`.
+**Publish topics**:
+- `rgb_topic`: a topic to publish the image from the simulator. Default is `/camera/rgb/image/compressed`
 
-**(Optional) Arguments/Parameters**:
-- `is_show_image`: whether to use `cv2.imshow` or not. Default is `False`
-- `queue_size`: the publish queue size. Default is `10`
+**Subscribe topics**:
+- `cmd_topic`: a topic to set driving info. Default is `/cmd_vel`
```

Comment:
I think they should be described as topics for the self-driving system side rather than the simulators side for more comprehending, i.e., people might misunderstand between `rgb_topic` and `dira_rgb_topic`.

Reply:
Agree.

How about the below description?

- `rgb_topic`: a topic to publish the image to the self-driving system. Default is `/camera/rgb/image/compressed`
- `cmd_topic`: a topic to set driving info sent from self-driving system. Default is `/cmd_vel`